NBitcoin/TransactionBuilder.cs

```diff
@@ -602,17 +607,18 @@ public bool DustPrevention
 		{
 			get;
 			set;
-		}
+		}
 
+		[Obsolete("Use Network.CreateTransactionBuilder(RBF) or ConsensusFactory.CreateTransactionBuilder(RBF) instead")]
```

Comment:
remove the obsolete, if the user change here, it is not big deal.

Reply:
fixed (and CI pipelines are green now)